README.md

```diff
@@ -53,4 +95,4 @@ This will update update the `Pipfile` and `Pipfile.lock` files, and install the
 
```

Comment:
I couldnt give feedback on the coding practices because there wasn't functional code. The markdown text was good through